src/yz_index_hashtree.erl

```diff
@@ -587,9 +588,8 @@ build_or_rehash(Tree, Locked, Type, #state{index=Index, trees=Trees}) ->
         {true, build} ->
             lager:debug("Starting YZ AAE tree build: ~p", [Index]),
             Indexes = yz_index:get_indexes_from_meta(),
-            InSync = yz_events:in_sync_with_metadata(),
```

Comment:
Of note in our *big* rebase, we should remove the old commit et. al, related to metadata sync.